e2e/package.json

```diff
@@ -5,18 +5,21 @@
     "open": "npx cypress open",
     "e2e": "npx cypress run",
     "open:dev": "CYPRESS_BASE_URL=http://localhost:4200 CYPRESS_BACKEND_URL=http://localhost:8080 npm run open",
-    "e2e:dev": "CYPRESS_BASE_URL=http://localhost:4200 CYPRESS_BACKEND_URL=http://localhost:8080 npm run e2e"
+    "e2e:dev": "CYPRESS_BASE_URL=http://localhost:4200 CYPRESS_BACKEND_URL=http://localhost:8080 npm run e2e",
+    "lint": "prettier --check cypress",
+    "format": "prettier --write cypress"
   },
   "private": true,
   "dependencies": {
     "debug": "^4.3.4",
     "jsonwebtoken": "^8.5.1",
     "mochawesome": "^7.1.3",
-    "typescript": "^4.7.4",
```

Comment:
Does docker compose run e2e work with typescript in the dev dependencies?

Reply:
As we don't ship a real app to a browser, we can misuse dev dependencies for deps that are already included in the cypress:included docker image. Therefore, moved typescript and prettier to the dependencies section.